assets/js/modules/analytics/components/setup/SetupForm.test.js

```diff
@@ -279,11 +314,6 @@ describe( 'SetupForm', () => {
 			body: {},
 		} );
 
-		fetchMock.get( getWebDataStreamsBatchRegexp, {
-			status: 200,
-			body: [],
-		} );
-
```

Comment:
removed as not needed, since we already populate the state with `webDataStreamsBatch` from fixtures.